pkg/build/build_test.go

```diff
@@ -0,0 +1,92 @@
+package build
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/suse-edge/edge-image-builder/pkg/config"
+)
+
+func TestPrepareBuildDir(t *testing.T) {
+	// Setup
+	bc := config.BuildConfig{}
+	builder := New(nil, &bc)
+
+	// Test
+	err := builder.prepareBuildDir()
+	defer os.Remove(bc.BuildTempDir)
+
+	// Verify
+	require.NoError(t, err)
+	_, err = os.Stat(bc.BuildTempDir)
+	require.NoError(t, err)
+	_, err = os.Stat(builder.combustionDir)
+	require.NoError(t, err)
+}
+
+func TestCleanUpBuildDirWithDelete(t *testing.T) {
+	// Setup
+	bc := config.BuildConfig{
+		DeleteArtifacts: true,
+	}
+	builder := New(nil, &bc)
+
+	testDir, err := os.MkdirTemp("", "eib-test-")
+	require.NoError(t, err)
+	bc.BuildTempDir = testDir
+
+	// Test
+	err = builder.cleanUpBuildDir()
+
+	// Verify
+	require.NoError(t, err)
+	_, err = os.Stat(bc.BuildTempDir)
+	require.Error(t, err)
+	require.True(t, os.IsNotExist(err))
```

Comment:
Nit: Can be `assert`. We usually only `require` the critical statements. Also applies to other tests in the same file.